tests/terraform/main.tf

```diff
@@ -219,7 +219,7 @@ resource "null_resource" "plugin_test" {
   depends_on = [null_resource.oci_private_key]
   provisioner "remote-exec" {
     inline = [
-      "NEEDRESTART_MODE=a DEBIAN_FRONTEND=noninteractive sudo -E apt-get install -y git golang make",
+      "NEEDRESTART_MODE=a DEBIAN_FRONTEND=noninteractive sudo -E apt-get update -y && sudo -E apt-get install -y git golang make",
```

Comment:
still tweaking this, but the acceptance tests pass with the bumped dependencies.